src/TheThingsNetwork.h

```diff
@@ -41,6 +46,7 @@ class TheThingsNetwork
   public:
     int downlinkPort;
     byte downlink[64];
+    TheThingsNetwork(fp_ttn_t fp);
```

Comment:
Isn't it confusing that we'd now have both a constructor argument and a separate `init()`? Can't we add it to `init()` or move all of them to individual setters so that `init()` doesn't grow into a big monster?